tests/Feature/Platform/Action/BuildGameListActionTest.php

```diff
@@ -307,58 +330,114 @@ public function testItCanSortByLastUpdated(): void
     }
 
     public function testItCanSortByReleasedAt(): void
+    {
+        $this->markTestSkipped('Has problems specifically with SQLite.');
```

Comment:
For whatever reason, while this test runs fine locally, it seems to fail in CI. Something must be different about SQLite in GitHub Actions.

Reply:
An alternative to skipping this altogether might be to detect if GitHub Actions is being used and skip it in that circumstance.

Reply:
Note that I'm now having this test skipped explicitly in CI only, as of https://github.com/RetroAchievements/RAWeb/pull/2706/commits/449bfb8d8fdc5ffb92b2f44e77b2cd20c619f590.